src/map/battle.cpp

```diff
@@ -3791,7 +3791,7 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 		case MO_EXTREMITYFIST:
 #ifdef RENEWAL
 			if (wd->miscflag&1)
-				skillratio += 100; // More than 5 spirit balls active
+				skillratio += 100 + 100 * (7 + sstatus->sp / 10); // More than 5 spirit balls active
```

Comment:
This is not the proper way to do it. I figured it how, however I don't know how to create a PR so I'll comment it here. Apologies for it.

Changed the code to this one.
`		case MO_EXTREMITYFIST:
			skillratio += 100 * (7 + sstatus->sp / 10);
#ifdef RENEWAL
			if (wd->miscflag&1)
				skillratio *= 2; // Doubles the damage by 100%
#endif			
			skillratio = min(500000,skillratio); //We stop at roughly 50k SP for overflow protection
			break;`

Reply:
I believe both codes are doing the same thing... I may change it tomorow... In your way is more elegant :)